cost/aws/instance_cloudwatch_utilization/AWS Inefficient Instance Utilization using CloudWatch.pt

```diff
@@ -1,10 +1,17 @@
-name "AWS Instance CloudWatch Utilization"
+name "AWS Inefficient Instance Utilization using CloudWatch"
 rs_pt_ver 20180301
 type "policy"
-short_description "Gathers AWS CloudWatch data for instances on 30 day intervals. See the [README](https://github.com/rightscale/policy_templates/tree/master/cost/aws/instance_cloudwatch_utilization/) and [docs.rightscale.com/policies](http://docs.rightscale.com/policies/) to learn more."
-long_description "Version: 1.5"
+short_description "This Policy Template gathers AWS instances with inefficient utilization using CloudWatch CPU and Memory Metrics over a 30 day average and downsized after approval. See the [README](https://github.com/rightscale/policy_templates/tree/master/cost/aws/instance_cloudwatch_utilization/) and [docs.rightscale.com/policies](http://docs.rightscale.com/policies/) to learn more."
```

Comment:
Please update the short_description 

**Checks for instances that are idle for the last 30 days and terminates them after approval.**
--

Reply:
It doesn't seem like this description is relevant to this policy, as I understand this policy is about resizing(downsize) underutilized EC2 instances. Still do I need to make the above change in the .pt file?

Reply:
Yes, that was my mistake.  your changes is good.